docs/advanced/vm-migration-network.md

```diff
@@ -0,0 +1,204 @@
+
+<Tabs>
+<TabItem value="ui" label="UI" default>
+
+:::tip
+
+Using the Harvester UI to configure the `vm-migration-network` setting is strongly recommended.
+
+:::
+
+#### Enable the VM Migration Network
+
+1. Go to **Advanced > Settings > vm-migration-network**.
+
+1. Select **Enabled**.
+
+1. Configure the **VLAN ID**, **Cluster Network**, **IP Range**, and **Exclude** fields to construct a Multus `NetworkAttachmentDefinition` CRD.
+
+1. Click **Save**.
+
+![vm-migration-network-enabled.png](/img/v1.6/vm-migration-network/vm-migration-network-enabled.png)
+
+#### Disable the VM Migration Network
+
+1. Go to **Advanced > Settings > vm-migration-network**.
+
+1. Select **Disabled**.
+
+1. Click **Save**.
+
+Once the VM migration network is disabled, KubeVirt starts using `mgmt` for VM migration-related operations.
+
+![vm-migration-network-disabled.png](/img/v1.6/vm-migration-network/vm-migration-network-disabled.png)
+
+</TabItem>
+
+<TabItem value="cli" label="CLI">
+
+You can use the following command to configure the [`vm-migration-network`](./settings.md#vm-migration-network) setting.
+
+```bash
+kubectl edit settings.harvesterhci.io vm-migration-network
+```
+
+The VM migration network is automatically enabled in the following situations:
+
+- The value field contains a valid JSON string.
+
+  ```yaml
+  apiVersion: harvesterhci.io/v1beta1
+  kind: Setting
+  metadata:
+    name: vm-migration-network
+  value: '{"vlan":100,"clusterNetwork":"vm-migration","range":"192.168.1.0/24", "exclude":["192.168.1.100/32"]}'
+  ```
+
+- The value field is empty.
+
+  ```yaml
+  apiVersion: harvesterhci.io/v1beta1
+  kind: Setting
+  metadata:
+    name: vm-migration-network
+  value: ''
+  ```
+
+The VM migration network is disabled when you remove the value field.
+
+  ```yaml
+  apiVersion: harvesterhci.io/v1beta1
+  kind: Setting
+  metadata:
+    name: vm-migration-network
+  ```
+
+:::caution
+
+Harvester considers extra insignificant characters in a JSON string as a different configuration.
+
+:::
+
+</TabItem>
+</Tabs>
+
+The following occur once the `vm-migration-network` setting is applied:
+
+- Harvester creates a new `NetworkAttachmentDefinition` and updates the KubeVirt configuration.
+- KubeVirt restarts all `virt-handler` pods to apply the new network configuration.
+
+### Post-Configuration Steps
+
+1. Verify that the setting's status is `True` and the type is `configured` using the following command:
+
+    ```bash
+    kubectl get settings.harvesterhci.io vm-migration-network -o yaml
+    ```
+
+    Example:
+
+    ```yaml
+    apiVersion: harvesterhci.io/v1beta1
+    kind: Setting
+    metadata:
+      annotations:
+        vm-migration-network.settings.harvesterhci.io/hash: ec8322fb6b741f94739cbb904fc73c3fda864d6d
+        vm-migration-network.settings.harvesterhci.io/net-attach-def: harvester-system/vm-migration-network-6flk7
+      creationTimestamp: "2022-10-13T06:36:39Z"
+      generation: 51
+      name: storage-network
```

Comment:
Should be vm migration network instead of `storage-network`?

Reply:
Updated it. Thanks.